src/cloud/docker/docker-containers-service.md

```diff
@@ -182,6 +182,14 @@ docker-compose run --rm redis redis-cli -h redis
 
 The Selenium container, based on the [selenium/standalone-chrome/](https://hub.docker.com/r/selenium/standalone-chrome/h), enables the [Magento Functional Testing Framework (MFTF)](https://devdocs.magento.com/mftf/docs/introduction.html) for Magento application testing in the Cloud Docker environment. See [Magento application testing]({{site.baseurl}}/cloud/docker/docker-test-app-mftf.html).
 
+## Test container
+
+**Container name**: test<br/>
```

Comment:
`<br/>` is not needed on these lines (187 - 189)

Reply:
All other containers description has `<br>`, without `<br>` text will be in one line.